datalab/mlalpha/_feature_slice_view.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+
+import json
+import pandas as pd
+from types import ModuleType
+
+import datalab.data
+import datalab.utils
+
+
+class FeatureSliceView(object):
+  """Represents A feature slice view."""
+
+  def _get_lantern_format(self, df):
+    if ('count' not in df) or ('feature' not in df):
+      raise Exception('No "count" or "feature" found in data.')
+    if len(df.columns) < 3:
+      raise Exception('Need at least one metrics column.')      
+    if len(df) == 0:
+      raise Exception('Data is empty')
+
+    metric_names = list(set(df) - set(['feature']))
+    data = []
+    for ii, row in df.iterrows():
+      metric_values = dict(row)
+      metric_values['count'] = metric_values['count']
```

Comment:
Added comments. Feature slice view browser expects the data in a certain format.